src/ontology/config/prefixes.jsonld

```diff
@@ -0,0 +1,30 @@
+{
+  "@context": {
+    "BAO": "http://www.bioassayontology.org/bao#BAO_",
+    "CoVIC": "https://cvdb.ontodev.com/antibody/",
+    "EFO": "http://www.ebi.ac.uk/efo/EFO_",
+    "ENM": "http://purl.enanomapper.org/onto/ENM_",
+    "ICD10CM": "http://purl.bioontology.org/ontology/ICD10CM/",
+    "ICD10WHO": "https://icd.who.int/browse10/2019/en#/",
+    "ICD11": "http://id.who.int/icd/entity/",
+    "IEDB": "http://iedb.org/",
+    "MEDDRA": "http://identifiers.org/meddra/",
+    "MEDGEN": "http://identifiers.org/medgen/",
+    "MESH": "http://identifiers.org/mesh/",
+    "NPO": "http://purl.bioontology.org/ontology/npo#NPO_",
+    "OMIM": "https://omim.org/entry/",
+    "OMIMPS": "https://omim.org/phenotypicSeries/PS",
+    "ONTIE": "https://ontology.iedb.org/ontology/ONTIE_",
+    "Orphanet": "http://www.orpha.net/ORDO/Orphanet_",
+    "PRO": "http://www.uniprot.org/annotation/PRO_",
+    "SCTID": "http://identifiers.org/snomedct/",
+    "UC": "http://purl.uniprot.org/core/",
+    "UMLS": "http://linkedlifedata.com/resource/umls/id/",
+    "UP": "http://purl.uniprot.org/uniprot/",
+    "faldo": "http://biohackathon.org/resource/faldo#",
```

Comment:
I forget exactly why we need this but I don't think that's super relevant for this PR! might be nice to tidy cruft some time but super low priority